qemu/tests/cfg/nvdimm.cfg

```diff
@@ -27,3 +28,11 @@
 
     variants:
         - nvdimm_basic:
+        - nvdimm_emulate:
+            # Please update host kernel line first, such as add "memmap=1G!5G":
```

Comment:
Please add this check in your test, like
try:
     process.system(grep 'memmap='  /proc/cmdline)
except process.CmdError:
     raise exceptions.TestError("Please add kernel param 'memmap' before start test.")